src/duckling/shell/file-toolbar.service.ts

```diff
@@ -0,0 +1,33 @@
+import {Injectable} from '@angular/core';
+
+/**
+ * Describes an action that is available from the file menus at the top of duckling.
+ */
```

Comment:
Can you provide comments describing each property? Or maybe just menuPath (such as the format of it)?